src/basic/process-util.c

```diff
@@ -1502,6 +1502,27 @@ int set_oom_score_adjust(int value) {
                                  WRITE_STRING_FILE_VERIFY_ON_FAILURE|WRITE_STRING_FILE_DISABLE_BUFFER);
 }
 
+int get_oom_score_adjust(int *ret) {
+        _cleanup_free_ char *t;
+        int r, a;
+
+        r = read_virtual_file("/proc/self/oom_score_adj", SIZE_MAX, &t, NULL);
+        if (r < 0)
+                return r;
+
+        delete_trailing_chars(t, WHITESPACE);
+
+        r = safe_atoi(t, &a);
+        if (r < 0)
+                return r;
+
+        assert(oom_score_adjust_is_valid(a));
```

Comment:
It seems inconsistent to return an error on syntax but assert on the value… If we assume the kernel never gives us invalid data, then we should assert on both. Otherwise, just return -ERANGE here.